tools/memap.py

```diff
@@ -98,44 +145,44 @@ def check_new_section_gcc(self, line):
         else:
             return False         # everything else, means no change in section
 
-    
-    def path_object_to_module_name(self, txt):
-        """ Parse a path to object file to extract it's module and object data
+
+    def parse_object_name_gcc(self, line):
+        """ Parse a path to object file
 
         Positional arguments:
         txt - the path to parse the object and module name from
         """
 
-        txt = txt.replace('\\', '/')
-        rex_mbed_os_name = r'^.+mbed-os\/(.+)\/(.+\.o)$'
-        test_rex_mbed_os_name = re.match(rex_mbed_os_name, txt)
+        line = line.replace('\\', '/')
+        rex_mbed_os_name = r'^.+\/(.+\.o)$'
```

Comment:
We have a precedent for the sort of style I'm suggesting: the `mbedToolchain` derived classes. I think that having one standard across the tools is generally easier on the reviewer. Further, I tend to assume that thing in all caps are not changing, which probably comes from the c preprocessor style.